sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ClientSideRequestStatistics.java

```diff
@@ -39,20 +44,25 @@
         DateTimeFormatter.ofPattern("dd MMM yyyy HH:mm:ss" + ".SSS").withLocale(Locale.US);
     private final static OperatingSystemMXBean mbean = (com.sun.management.OperatingSystemMXBean)ManagementFactory.getOperatingSystemMXBean();
 
-    private final ZonedDateTime requestStartTime;
-    private ZonedDateTime requestEndTime;
+    private final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
`ObjectMapper` instantiation is very expensive, and this is instantiated per request.

Reply:
Moved to static